drivers/i2c/i2c_ite_it8xxx2.c

```diff
@@ -262,61 +201,22 @@ static void i2c_standard_port_set_frequency(const struct device *dev,
 	 * for setting. So we can adjust tlow to meet timing.
 	 * The others use basic 50/100/1000 KHz setting.
 	 */
-	if (freq_khz == 400) {
+	if (freq_khz == I2C_BITRATE_FAST) {
```

Comment:
The `freq` parameter is now in Hz, not kHz.  Please update.

Reply:
Done.